nrrd/writer.py

```diff
@@ -124,6 +124,10 @@ def write(filename, data, header={}, detached_header=False, custom_field_map=Non
     custom_field_map : :class:`dict` (:class:`str`, :class:`str`), optional
         Dictionary used for parsing custom field types where the key is the custom field name and the value is a
         string identifying datatype for the custom field.
+    compression_level : int
+        Int specifying compression level, when applicable.
```

Comment:
Instead of *when applicable*, maybe be more specific and say *when encoding is set to compression for BZIP or GZIP*

Reply:
Ok